src/test/java/org/mastodon/mamut/clustering/ui/DendrogramPanelDemo.java

```diff
@@ -0,0 +1,49 @@
+package org.mastodon.mamut.clustering.ui;
+
+import com.apporiented.algorithm.clustering.AverageLinkageStrategy;
+import com.apporiented.algorithm.clustering.Cluster;
+import com.apporiented.algorithm.clustering.ClusteringAlgorithm;
+import com.apporiented.algorithm.clustering.DefaultClusteringAlgorithm;
+import org.mastodon.mamut.clustering.util.Classification;
+
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+import javax.swing.WindowConstants;
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.util.Collections;
+
+public class DendrogramPanelDemo
```

Comment:
A javadoc comment that describes what the class does would be awesome

Reply:
Done here: https://github.com/mastodon-sc/mastodon-deep-lineage/commit/bc81b778832ee031aad1f465ae773754cef10acb